docs/fulltext.adoc

```diff
@@ -252,3 +252,29 @@ Again we're search for "Johnny", expecting now two of them:
 CALL apoc.index.search("people", "Johnny") YIELD node, weight
 RETURN node.name, weight
 ----
+
+=== Fulltext Index and multilingual string
+By default standart configuration of fultext index use standart analyzer for extract token from text.
+For support fultext search on multilangual content we need to set additional configuration to parameter `<option>` in function `apoc.index.addAllNodes`.
+
+Make sure `apoc.multilingual.enabled=true` is set.
```

Comment:
where is the defined in the code?

Reply:
I write "apoc.multilingual.enabled=true" if you decide to make multilingual index optional

Reply:
But where is the code that uses it?

Reply:
There is no code that use it.